api/app/lib/updaters.rb

```diff
@@ -102,7 +102,7 @@ def update_belongs_to(model, name, to_add)
   def update_has_many(model, name, to_add)
     value = relationship_map(model, name, to_add)
     model.send("#{name}=", value)
-    model.send(name.to_s).try(:sort, value)
+    model.send(name.to_s).try(:sort_makers, value)
```

Comment:
Or perhaps we should add a post update relationship hook of some kind. Happy to discuss.